### DISCUSSION TITLE
Collapsible Menu and ToC

### DISCUSSION THREAD

**Discussion by jaceklaskowski** [squidfunk/mkdocs-material#2196] [Ideas]
Just stumbled upon [this page](https://docs.gigaspaces.com/15.8/admin/kubernetes-spark-job.html) with collapsible menu and ToC. Is this possible in the current version? Possible in the future?

<img width="1680" alt="material-collapsible-menu-toc" src="https://user-images.githubusercontent.com/62313/104102737-84210900-529e-11eb-9a4d-2867343beecf.png">

**Comment by squidfunk** [comment]
This should definitely be possible, but I'm undecided on how it should look with the Material theme. Both sidebars are free-floating, there's no "handle" where to put the button on. However, I think this is a good candidate for a feature request, so we could discuss this with other users. Would you like to open a feature request?

**Reply by hellt** [reply]
was it ever discussed given the age of that idea?
I often find myself in need for a collapsible toc/scrollspy and would like to share ideas

**Reply by squidfunk** [reply]
Previous discussion in #3428 and #4272. Not much interest among users, so currently no plans to implement.
